### ISSUE TITLE
Returning NoneType error when getting particular artist lyrics

### ISSUE THREAD

**Issue by ethan-ou** [jameswenzel/pylyrics3#7]
I'm getting an error when getting artist lyrics with some artists, but not all of them. For instance, Taylor Swift, Paramore and Charlie Puth work fine, but Troye Sivan, Jessie Ware and Kina Grannis return this error:

```
Traceback (most recent call last):
  File "download-lyrics.py", line 42, in <module>
    artist = pylyrics3.get_artist_lyrics(artist_name)
  File "C:\Python37\lib\site-packages\pylyrics3.py", line 47, in get_artist_lyrics
    song_urls = self.__get_artist_song_links(soup)
  File "C:\Python37\lib\site-packages\pylyrics3.py", line 204, in __get_artist_song_links
    link = PyLyrics3.__lyric_wiki_url + link_tag.get('href')
TypeError: can only concatenate str (not "NoneType") to str
```

Do you have any clue why?

**Comment by emo-eth** [comment]
Hey Ethan,

Thanks for bringing this up. Some links didn't have `href` attributes, which is strange.
In looking into this, I found some other issues, namely that compilation and collaborative songs weren't included in the response (`Strawberries & Cigarettes` and `1999` in Troye's case).
I've fixed both problems and released a new version, `1.0.6` on pypi. 
Give it an install with `pip install pylyrics3 --upgrade` and let me know if you run into other issues.

Thanks!
